spec/haskell/src/SEL4/Machine/Hardware/ARM.lhs

```diff
@@ -21,7 +21,7 @@ This module defines the low-level ARM hardware interface.
 
 > import Control.Monad.Reader
 > import Data.Bits
-> import Data.Word(Word8, Word32, Word64)
+> import Data.Word(Word8, Word16, Word32, Word64)
```

Comment:
I'm seeing Word16 added in a bunch of includes, but you don't appear to be using it.

Reply:
good catch, they need to go (the first implementation used Word16 and it looks like I didn't catch all of them when I adapted it).